src/docs/devices/M5Stack-M5StickC-PLUS/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: M5Stack M5StickC PLUS
+date-published: 2023-10-16
+type: misc
+standard: global
+board: esp32
+project-url: https://github.com/martydingo/esphome-axp192
```

Comment:
While this is a part of it, I wouldn't say its the project-url...
Might be better to link to m5stack official doc page for this device